app/styles/_core.less

```diff
@@ -466,41 +466,77 @@ label.checkbox {
 
 .osc-secrets-form {
   .basic-secrets {
```

Comment:
I'd tend to suggest moving it out of `_core.less` and into `_secrets.less` or something as well.